src/picknik_ur_multi_arm_config/config/config.yaml

```diff
@@ -112,7 +112,7 @@ ros_global_params:
 # [Required]
 moveit_params:
   # Used by the Waypoint Manager to save joint states from this joint group.
-  joint_group_name: "multi_arm_manipulator"
+  joint_group_name: "first_manipulator"
```

Comment:
this group gets used by imarker or pose jog, so I'm guessing we have to use a group with a tip_frame

Reply:
Yes, that is my understanding too